config/locales/inherited_proofing/fr.yml

```diff
@@ -5,6 +5,7 @@ fr:
       continue: Continuer
     headings:
       lets_go: to be implemented
+      retrieval: Nous récupérons vos informations depuis My HealtheVet.
```

Comment:
The other languages don't include a period at the end of this sentence. Should they be consistent?

Reply:
I defer to the designers on this one.